dust_extinction/grain_models.py

```diff
@@ -606,3 +606,85 @@ def __init__(self, modelname="MWRV31", **kwargs):
         self.data_tolerance = 1e-6
 
         super().__init__(**kwargs)
+
+
+class Y24(BaseExtGrainModel):
+    r"""
+    Ysard et al. (2024) Grain Model
```

Comment:
Might be worth mentioning that this is THEMIS 2.0, to avoid confusion in the future.